frontend/src/views/IssueDetail.vue

```diff
@@ -892,19 +892,41 @@ export default defineComponent({
     };
 
     const currentPipelineType = computed((): PipelineType => {
+      if (state.create) {
+        if (issue.value.type == "bb.issue.database.create") {
+          return "NO_PIPELINE";
+        } else if (issue.value.type == "bb.issue.database.schema.update") {
+          const createContext = ((issue.value as IssueCreate).createContext) as UpdateSchemaContext;
+          if (createContext.updateSchemaDetailList.length > 1) {
+            return "MULTI_STAGE";
+          }
+          return "NO_PIPELINE";
+        }
+      }
       return pipelineType(issue.value.pipeline);
     });
 
-    const selectedStage = computed((): Stage | StageCreate => {
+    const selectedStage = computed((): Stage | UpdateSchemaDetail => {
```

Comment:
I am afraid this is confusing, selectedStage returns a UpdateSchemaDetail type, I feel it's more readable to repurpose StageCreate to embed the required UpdateSchemaDetail or maybe have StageClientCreate StageServerCreate separately.